spec/unit/network/http/connection_spec.rb

```diff
@@ -42,31 +42,18 @@
     end
   end
 
-  context "when methods that accept a block are called with a block" do
-    let (:host) { "my_server" }
+  context "when methods that accept a block are called with a block", :vcr => true do
```

Comment:
That's valuable feedback, thanks Kylo. I'm all :+1: for being more in keeping with common documentation.